core/src/navigation/TopNav.html

```diff
@@ -1,176 +1,181 @@
 <svelte:window on:click="closeAllDropdowns()" on:blur="closeAllDropdowns()" />
-
-<div class="fd-shellbar {hideNavComponent ? 'hideNavComponent' : ''}">
-  <div class="fd-shellbar__group fd-shellbar__group--start">
-    <LogoTitle />
-  </div>
-  <div class="fd-shellbar__group fd-shellbar__group--end">
-    <div class="fd-shellbar__actions">
-      {#if !authorizationEnabled || isLoggedIn}
-      <ContextSwitcher
-        bind:dropDownStates
-        on:toggleDropdownState="toggleDropdownState(event.name)"
-      />
-      {/if} 
-      {#if children && pathData.length > 0} 
-        {#each children as node} 
-          {#if !node.hideFromNav}
-          <div
-            class="fd-shellbar__action fd-shellbar__action--hide fd-shellbar__action--collapsible"
-          >
-            <button
-              title="{node.label}"
-              class="fd-button--shell {node === selectedNode ? 'is-selected' : ''} {node.externalLink && node.externalLink.url ? 'fd-global-nav__btn--external' : ''}"
-              aria-controls="0AcWE812"
-              aria-expanded="false"
-              aria-haspopup="true"
-              on:click="handleClick(node)"
+{#if authorizationEnabled && (!autologinEnabled || isLoggedIn) || !authorizationEnabled }
```

Comment:
I would add the extra parenthesis to avoid anyone reading the code the extra thinking:
```{#if (authorizationEnabled && (!autologinEnabled || isLoggedIn)) || !authorizationEnabled }```
Additionally, I would set a variable for that and use it here and below, to make explicit the relation between this value and the input used for authorization component:
```
{#if showTopNav }
...
...
<Authorization isHidden={!showTopNav}/> // or <Authorization showMarkup={showTopNav}/> which is simpler
```

Reply:
The second thing is impossible because in most cases Authorization is displayed inside TopNav which should be hidden in some of them. So it's logically impossible :)

Reply:
I meant in the top level if statement, where you used: `<Authorization isHidden={true}/>`. Correct me if I am wrong, but I think it should be possible to use the variable there.

Reply:
Well, that would change nothing at all except the fact of having `{!showTopNav}` instead of `{true}`. If this particular <Authorization> is used, it means that `showTopNav` is false. I can change it if you want but for me `{false}` is more clear here - it indicates the fact I mentioned (it is always false here).

Reply:
I see your point. I am fine keeping it as it is.